kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -18,9 +18,15 @@ object StubRenderer {
             val kDoc = if (shouldExportKDoc) {
                 descriptor?.extractKDocString()?.let {
                     if (it.isNotEmpty()) {  // sometimes `findDoc` return empty string; is it a bug?
+                        assert(it.startsWith("/**") && it.endsWith("*/"))
```

Comment:
Why add the assert here, if the code below checks the same below and gracefully handles the case when it is not true?